site/content/blog/2020-06-04-zap-2-9-0-highlights/index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "ZAP 2.9 Highlights"
+description: "Highlights from ZAP release 2.9.0"
+summary: "Do you know what interesting bits were added to ZAP 2.9.0? Don't read release notes? This blog post is for you! 
+Session Management Scripts, Proxy Info Display, Proxy Port Reservation Failure Handling, Options Panel(s) Filter, Active Scan Filter, and more."
+images:
+- https://www.zaproxy.org/blog/2020-06-04-zap-2-9-0-highlights/images/zapbot.png
+type: post
+tags:
+- blog
+- youdontknowzap
+date: "2020-06-04"
+addSocialPreview: true
+authors:
+    - thorin
+---
+
+Do you know what interesting bits were added to ZAP 2.9.0?
+Don't read release notes?
+
+This blog post is for you!
+
+## Proxy Info Display
+
+Contributed by: [Rick Mitchell (kingthorin)](https://twitter.com/kingthorin_rm).
+
+The primary ZAP proxy is now displayed in the left footer area of the main window:
+
+![Primary Proxy Display in Left Footer Area](images/footer_proxy.png)
+
+Futher if you have additional proxies setup, they'll be displayed in the tooltip text when you hover your cursor over the primary proxy display:
+
+![Local Proxy Options](images/local_proxy_options.png)
+
+![Proxy Tooltip Display](images/footer_proxy_tooltip.png)
+
+## Proxy Port Reservation Failure Handling
+
+Contributed by: [Rick Mitchell (kingthorin)](https://twitter.com/kingthorin_rm).
+
+Have you ever had ZAP complain at startup that it can't bind to the default port (8080), because you're running Tomcat (or something else) locally? No problem
+ZAP will now suggest a port that you could use.
+
+![Animated GIF illustrating the startup behavior when a proxy port conflict is encountered](images/startup_conflict.gif)
+
+## Options Panel(s) Filter
+
+Contributed by: [Dennis Kniep](https://twitter.com/dennis_kniep)
+
+ZAP has a quite extensive options interface, sometimes it's not straight forward to locate the setting you want/need. Now you can search settings
+and have things highlighted for you:
+
+In the following example the user searched the Options panel for "proxy", you can see all the panels names outlined in yellow where the panel contains the word "proxy"
+
+![Options Panel "Proxy" Search Example](images/options_search.png)
+
+Selecting one of those panels the applicable labels/fields will be highlighted:
+
+![Options Panel "Proxy" Search Results](images/options_search_result.png)
+
+This functionality is available in multiple places including the Active Scan Policy interface:
+
+![Active Scan Policy Search Example](images/scan_policy_search.png)
+
+![Active Scan Policy Search Result](images/scan_policy_search_result.png)
+
+## Active Scan Filter
+
+Contributed by: [Karan Preet Singh Sasan (@preetkaran20)](https://github.com/preetkaran20).
+
+When starting an active scan users can now define filter criteria for inclusion/exclusion of various requests. The available criteria include:
```

Comment:
Maybe add a comment about how flexible the ZAP active scan is anyway, but that this makes it even more flexible?